### PR TITLE
create EventQuadripoleDisconnection model

### PR DESCRIPTION
**Pull request by marcosmc**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Feature

**What is the new behavior (if this is a feature change)?**
Add the extension to load EventQuadripoleDisconnection model

### CODE REVIEW

File: src/main/groovy/com/powsybl/dynawo/events/EventQuadripoleDisconnectionGroovyExtension.groovy [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    }
+
+    String getName() {
+        return "Dynawo"
+    }
+    
+    void load(Binding binding, Consumer<EventModel> consumer) {
+        binding.EventQuadripoleDisconnection = { Closure<Void> closure ->
+            def cloned = closure.clone()
+            EventQuadripoleDisconnectionSpec cventQuadripoleDisconnectionSpec = new EventQuadripoleDisconnectionSpec()
```
Comment (agnesLeroy) [inline src/main/groovy/com/powsybl/dynawo/events/EventQuadripoleDisconnectionGroovyExtension.groovy:50]: I think I'd rather have this variable be named eventQuadripoleDisconnectionSpec

File: src/main/java/com/powsybl/dynawo/events/EventQuadripoleDisconnection.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+import com.powsybl.dynawo.xml.DynawoXmlContext;
+import com.powsybl.dynawo.xml.MacroConnectorXml;
+
+/**
+ * @author Marcos de Miguel <demiguelm at aia.es>
+ */
+public class EventQuadripoleDisconnection extends AbstractBlackBoxEventModel {
+
+    public EventQuadripoleDisconnection(String dynamicModelId, String staticId, String parameterSetId) {
```
Comment (agnesLeroy) [inline src/main/java/com/powsybl/dynawo/events/EventQuadripoleDisconnection.java:24]: Here are you sure it's dynamicModelId, and not eventModelId?

File: src/main/groovy/com/powsybl/dynawo/events/EventQuadripoleDisconnectionGroovyExtension.groovy [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+            this.staticId = staticId
+        }
+
+        void parameterSetId(String parameterSetId) {
+            this.parameterSetId = parameterSetId
+        }
+    }
+
+    String getName() {
+        return "Dynawo"
```
Comment (agnesLeroy) [inline src/main/groovy/com/powsybl/dynawo/events/EventQuadripoleDisconnectionGroovyExtension.groovy:44]: Here, why do you return Dynawo? Shouldn't it be more specific?

Comment (marcosmc) [reply src/main/groovy/com/powsybl/dynawo/events/EventQuadripoleDisconnectionGroovyExtension.groovy:44]: This method is used to load all the extensions of a specific dynamic simulator type, in this case we want to obtain all the extensions from Dynawo, that is why all the extensions of this project return the text "Dynawo" in this method.

File: src/main/java/com/powsybl/dynawo/events/EventQuadripoleDisconnection.java (hunk 2 above)
Comment (marcosmc) [reply src/main/java/com/powsybl/dynawo/events/EventQuadripoleDisconnection.java]: Done

File: src/main/groovy/com/powsybl/dynawo/events/EventQuadripoleDisconnectionGroovyExtension.groovy (hunk 3 above)
Comment (agnesLeroy) [reply src/main/groovy/com/powsybl/dynawo/events/EventQuadripoleDisconnectionGroovyExtension.groovy:44]: oh ok, thanks

File: src/main/groovy/com/powsybl/dynawo/events/EventQuadripoleDisconnectionGroovyExtension.groovy (hunk 1 above)
Comment (marcosmc) [reply src/main/groovy/com/powsybl/dynawo/events/EventQuadripoleDisconnectionGroovyExtension.groovy]: Done

**Review by agnesLeroy — APPROVED** [review]

**Pull request merged**